corehq/apps/locations/views.py

```diff
@@ -260,6 +260,11 @@ class LocationOptionsController(EmwfOptionsController):
     namespace_locations = False
     case_sharing_only = False
 
+    def __init__(self, *args):
+        args = list(args)
+        self.include_locations_with_no_users_allowed = args.pop(3)
```

Comment:
nit: I think it would be more future proof/ less future error prone to just make this a keyword argument